examples/aci-linux-multi/main.tf

```diff
@@ -0,0 +1,30 @@
+resource "azurerm_resource_group" "aci-rg" {
+    name="aci-test"
+    location="west us"
+}
+
+resource "azurerm_container_group" "aci-test" {
+    name = "mynginx"
+    location = "west us"
```

Comment:
_minor_ since Location is available on the Resource Group resource, could we update this to `"${azurerm_resource_group.aci-rg.location}"` to keep it a little DRY-er?, rather than in each resource? (I'm slowly working through the documentation for each resource to show this)

Reply:
Done.